recipes/pwlf/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: pwlf
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pwlf/pwlf-{{ version }}.tar.gz
+  sha256: d4756dfcd89d9182198c52995dc8b37299be509bf4742b0b1378385e133682dd
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - pydoe >=0.3.8
+    - numpy
+    - scipy >=1.2.0
+    - setuptools
```

Comment:
See [this](https://github.com/conda-forge/staged-recipes/pull/11336#issuecomment-617163048) comment.